web-client/src/app/services/scanner.service.ts

```diff
@@ -7,6 +7,7 @@ import { Camera, CameraPermissionState } from '@capacitor/camera';
 export class ScannerService {
   constructor() {}
 
+  /* istanbul ignore next not sure if there is a way to easily spy on Camera.checkPermissions() */
```

Comment:
I saw [this](https://capacitorjs.com/docs/guides/mocking-plugins) yesterday, not sure if it's relevant

Reply:
Aaah, that seems to be the way to do it. I think it make sense to have this go in separately for now. Created an issue #62